rpcs3/Emu/System.cpp

```diff
@@ -1659,38 +1659,14 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 
 			m_force_boot = false;
 		}
-		else if (IsPaused())
-		{
-			m_state = system_state::ready;
-			GetCallbacks().on_ready();
-		}
+
 		return game_boot_result::no_errors;
 	}
 }
 
 void Emulator::Run(bool start_playtime)
 {
-	if (!IsReady())
-	{
-		// Reload with prior configuration.
-		Load(m_title_id, false, m_force_global_config);
```

Comment:
Why remove this? This looks like you don't know why it even exists

Reply:
It's not used anywhere. I checked.

Reply:
Maybe you mix with Emulator::Stop(true) which does similar thing.

Reply:
hmm. Looks like this code is essentially 8 years old, and only got improved for the sake of compliance.
I wonder when it was actually last used.